modules/acquia_cms_tour/acquia_cms_tour.module

```diff
@@ -57,12 +57,12 @@ function acquia_cms_tour_modules_uninstalled(array $modules) {
 }
 
 /**
- * Implements hook_content_model_role_presave_alter().
+ * Implements hook_cms_tour_entity_insert().
  */
-function acquia_cms_tour_content_model_role_presave_alter(RoleInterface &$role) {
-  switch ($role->id()) {
+function acquia_cms_tour_entity_insert(EntityInterface $entity) {
+  switch ($entity->id()) {
     case 'content_administrator':
-      $role->grantPermission('access acquia cms tour dashboard');
+      user_role_grant_permissions('content_administrator', ['access acquia cms tour dashboard']);
```

Comment:
Why passing hard-coded role name?

Reply:
We are assigning permissions exclusively to the role with the matching ID `content_administrator`.